simulation/src/test/java/com/palantir/dialogue/core/SimulationTest.java

```diff
@@ -119,11 +118,10 @@ public void simplest_possible_case() {
                         .handler(h -> h.response(200).responseTime(Duration.ofMillis(1000)))
                         .build());
 
-        Channel channel = strategy.getChannel(simulation, servers);
         result = Benchmark.builder()
-                .requestsPerSecond(50)
-                .sendUntil(Duration.ofSeconds(20))
-                .channel(channel)
+                .requestsPerSecond(11)
+                .sendUntil(Duration.ofMinutes(20))
```

Comment:
I dialled this up so we'd hit the 10 minute reshuffle in the pin until error strategy